src/data/iconData.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
There was a reason the track talked about filtering the icon value to be of 200, 300, 400 etc, if you look at the URLs here all the 2xx ones are the same, as are all the 3xx ones etc. Theres no need to import all this data if using the string method to boil it down just to what you need